dora/core/server/master/src/main/java/alluxio/master/job/DoraLoadJob.java

```diff
@@ -225,28 +185,34 @@ public URIStatus next() {
    */
   public List<DoraLoadTask> prepareNextTasks() {
     LOG.debug("Preparing next set of tasks for jobId:{}", mJobId);
-    ImmutableList.Builder<URIStatus> batchBuilder = ImmutableList.builder();
+    ImmutableList.Builder<UfsStatus> batchBuilder = ImmutableList.builder();
     int i = 0;
     int startRetryListSize = mRetryFiles.size();
     int filesToLoad = 0;
     while (filesToLoad < RETRY_THRESHOLD
-        && i++ < startRetryListSize
-        && mRetryFiles.peek() != null) {
+        && i++ < startRetryListSize && mRetryFiles.peek() != null) {
       String path = mRetryFiles.poll();
       try {
-        URIStatus uriStatus = mFs.getStatus(new AlluxioURI(path));
+        UfsStatus uriStatus = mUfs.getStatus(path);
```

Comment:
actually, we can deal with FileNotFoundException when we process the response, so we can avoid extra getStatus calls which could be expensive.